code/modules/client/preferences.dm

```diff
@@ -1868,6 +1872,14 @@ GLOBAL_LIST_INIT(food, list( // Skyrat addition
 					var/msg = stripped_multiline_input(usr, "Set your OOC Notes", "OOC Notes", skyrat_ooc_notes, MAX_FLAVOR_LEN, TRUE)
 					if(msg)
 						skyrat_ooc_notes = html_decode(msg)
+				
+				if("bloodtype")
+					var/msg = input(usr, "Choose your blood type", "Blood Type", "") as anything in (pref_species.bloodtypes + "Default")
+					if(msg)
```

Comment:
etc

Reply:
it's in the skyrat change section its good

Reply:
mb